### DISCUSSION TITLE
Unit 1.4 Part 4 - Error

### DISCUSSION THREAD

**Discussion by gsingh10000** [Lightning-AI/dl-fundamentals#22] [General discussions]
At time 3:19 onwards, the example computing the weighted sums  has the x1, x2 values flipped.
<img width="720" alt="Screen Shot 2023-02-07 at 12 18 19 AM" src="https://user-images.githubusercontent.com/79126179/217155433-dbf144e4-b033-4971-8f36-7be2057c07c1.png">

**Comment by rasbt** [comment]
Thanks a lot for the note! I will add that to the errata list for now, and after finishing processing the current units, we will swap the slides in the video to fix that!

**Comment by rasbt** [comment]
It's in the errata and we will swap the slides soon to address that. Closing it here for some spring cleaning :)
